testsuite/test-modprobe.c

```diff
@@ -47,6 +47,14 @@ DEFINE_TEST(modprobe_show_depends2,
 	.output = {
 		.out = TESTSUITE_ROOTFS "test-modprobe/show-depends/correct-mod-simple.txt",
 	});
+DEFINE_TEST_WITH_FUNC(modprobe_show_depends_no_load, modprobe_show_depends2,
```

Comment:
I wonder what the extra files were meant for. Glancing around:

It seems like modprobe has been using `dry-run` (aka don't load the module) and `ignore-loaded` (aka don't look at /proc or /sys) since day one... Where the porting commit eeb627004b6ee3610913f98ef143e84eb0546360 added these seemingly unused files.

AFAICT the extra test looks good so :+1: from me.